gstreamer/src/event.rs

```diff
@@ -1667,3 +1667,27 @@ impl<'a> CustomBothOobBuilder<'a> {
         ev
     });
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_simple() {
+        ::init().unwrap();
+
+        // Event without arguments
+        let flush_start_evt = Event::new_flush_start().build();
+        match flush_start_evt.view() {
+            EventView::FlushStart(_) => (),
+            _ => panic!("flush_start_evt.view() is not an EventView::FlushStart(_)"),
+        }
+
+        // Event with arguments
+        let flush_stop_evt = Event::new_flush_stop(true).build();
+        match flush_stop_evt.view() {
+            EventView::FlushStop(flush_stop_evt) => assert!(flush_stop_evt.get_reset_time()),
```

Comment:
What exactly is this assertion doing?

Reply:
Checking that the `reset_time` is `true` as it was initialized when building the event. This is on par with `assert_eq!(buffering_msg.get_percent(), 42)`.

I can make it more explicit like `assert_eq!(flush_stop_evt.get_reset_time(), true)` if you prefer.

Reply:
Oh it's a boolean, not a time :) right. That's all OK then